gitrevise/merge.py

```diff
@@ -28,6 +28,12 @@
 DEFAULT_CONFLICT_MARKER_SIZE = 7
 
 
+def get_conflict_marker_size(__repo: Repository, __file: Path) -> int:
```

Comment:
this should maybe be a method of Repository (or at least the generic check-attr wrapper should be)

Reply:
> or at least the generic check-attr wrapper should be
Yep, I agree that a general check-attr wrapper thing would benefit from being lifted up, if implemented. It would make sense to allow that to be a persisted process and have those path → attributes cached.

As far as handling the `"conflict-marker-size"` attribute in particular, it might benefit from staying standalone. There's some extra validation vs other attributes (turning it into an int, and defaulting to 7 if non-positive), and it's very specific to this area of code dealing with merge-conflict text.